### PR TITLE
Create migrating-data.liquid

### PR DESCRIPTION
**Pull request by diana-lakatos**
@ignacy I wrote a tutorial for Migrating Data based on your description. Please review!
I think it would be best to add the example you had as a screenshot in your original description as a code sample in the tutorial. Could you please provide that? Also, if available, it would be helpful to add a (couple of) code sample(s) to Step 2.

### CODE REVIEW

File: marketplace_builder/views/pages/how-to/migrations/migrating-data.liquid [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Migrating data is a three-step process:
+
+1. Generate empty migration
+2. Implement migration
+3. Run migration
+
+### Step 1: Generate empty migration
+Generate an empty migration using the `marketplace-kit migrations` command: 
+
+<pre class="command-line" data-output="2-50" data-user="user" data-host="host"><code class="language-bash">
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/migrations/migrating-data.liquid]: `data-output` value (2-50) marks lines from - to where there is an output of a shell command - in this case there is no output at all (only user input), so you can remove the attribute altogether

Comment (diana-lakatos) [reply marketplace_builder/views/pages/how-to/migrations/migrating-data.liquid]: @pavelloz Thank you! Fixed.

**Comment by ignacy** [comment]
@diana-lakatos I see that step 1 & 3 already have the command in them, like in step1:

```
 marketplace-kit migrations generate ENVIRONMENT MIGRATION_NAME 
```

I'm not sure what would be a good example for that?

```
 marketplace-kit migrations generate local add_some_books
```
Which will create a file with a name like `20180909123423_add_some_books.liquid`

Is this useful? Or is the generic version clear enough? 

For step2 we can add an example of a mutation that can be put into the file created by the command in step1, for example like this:

```liquid
 {% execute_query 'create_book', name: "Animal Farm", author: "George Orwell" %}
```

For step3 we could follow what was done for step 1 and say:

```
marketplace-kit migrations run local 20180909123423
```

**Comment by diana-lakatos** [comment]
@ignacy Thank you! I updated the tutorial with your examples, please review! Thank you!

**Review by ignacy — APPROVED** [review]
:+1: LGTM :shipit:

**Pull request merged**